### PR TITLE
Create jamied132.json

### PR DESCRIPTION
**Pull request by JamieD132**
Yay I wanna get a jamied132.is-truly.cool domain
Wait am I the first person to do this???

### CODE REVIEW

File: domains/jamied132.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+  "owner": {
+    "username": "JamieD132",
+    "email": "cheese@jamied132.is-a.dev",
+    "darflen": "JamieD132"
+  },
+  "record": {
+    "CNAME": "jamied132.github.io"
```
Comment (CuteDog5695) [inline domains/jamied132.json:8]: I don't think you can add 2 domains in a single repo (if your goal is to use your site along with the is-a.dev one)

**Review by CuteDog5695 — CHANGES_REQUESTED** [review]

**Comment by CuteDog5695** [comment]
You could probably like use redirect.pizza to redirect this domain to the is-a.dev one.

File: domains/jamied132.json (hunk 1 above)
Comment (JamieD132) [reply domains/jamied132.json:8]: I was planning on connecting my is-a.dev one to cloudflare pages instead and this one to github

Comment (CuteDog5695) [reply domains/jamied132.json:8]: oh, should be fine then

**Review by CuteDog5695 — APPROVED** [review]
Seems good to me now

**Comment by CuteDog5695** [comment]
>Wait am I the first person to do this???

First non-maintainer, ye :D

**Comment by CuteDog5695** [comment]
Just found out the root domain was invalid

**Comment by CuteDog5695** [comment]
I'll merge it even in checks failed since root file was the cause.

**Pull request merged**